propagation_test.go

```diff
@@ -148,6 +148,42 @@ func TestBaggagePropagationHTTP(t *testing.T) {
 	assert.Equal(t, map[string]string{"some-key": "98:765"}, sp2.(SpanContext).baggage)
 }
 
+func TestJaegerBaggageHeader(t *testing.T) {
+	tracer, closer := NewTracer("hobbits", NewConstSampler(true), NewNullReporter())
```

Comment:
that's precisely the attitude. The joke itself is perfectly acceptable for string literals. But we don't quote the humorless lord of the nose rings in our (grown up) code.